docs/dev-guide/design-details.md

```diff
@@ -205,7 +205,7 @@ It should also allow for the following actions:
 - single and batch re-processing of transactions
 - querying for monitoring statistics
 
-The API reference as it currently exists can be [found here](./dev-guide/api-ref.html).
+The API reference as it currently exists can be [found here](../api/transactions/overview).
 
 ### API Authentication
```

Comment:
@BMartinos 
Do you know if we have any updated docs for this section? The issue referenced below is closed

Reply:
Lets remove this for now, we can always bring it back in if its needed